SIL.WritingSystems.Tests/SldrTests.cs

```diff
@@ -444,5 +444,31 @@ public void LanguageTags_NoCachedLangTagsSldrOffline_UseEmbeddedLangTags()
 				Assert.That(File.Exists(langTagsPath), Is.False);
 			}
 		}
+
+		[Test]
+		public void GetUnicodeCategoryBasedOnICU_HighSurrogate_GetsCorrectCategory()
+		{
+			Assert.That(Sldr.GetUnicodeCategoryBasedOnICU("\U00020056\U000200D9", 0),
+				Is.EqualTo(UnicodeCategory.OtherLetter));
+			Assert.That(Sldr.GetUnicodeCategoryBasedOnICU("\U00020056\U000200D9", 2),
```

Comment:
I personally would prefer to make this two test cases. That way it's clearer that we're testing two scenarios instead of one scenario with two asserts.